doc/conf.py

```diff
@@ -1,95 +1,87 @@
-
-module_setup = imp.load_source('module_setup', os.path.join(os.path.dirname(__file__), '..', '..', 'setup.py'))
-VERSION = module_setup.VERSION
-AUTHOR = module_setup.AUTHOR
-DOCUMENTATION_NAME = module_setup.DOCUMENTATION_NAME
-
-TITLE = "%s Documentation v%s" % (DOCUMENTATION_NAME, VERSION)
-
+import sys
+import os
+import shlex
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..'))
-import orangecontrib.network
-sys.modules["Orange.network"] = orangecontrib.network
+#sys.path.insert(0, os.path.abspath('.'))
 
-# -- General configuration -----------------------------------------------------
+# -- General configuration ------------------------------------------------
 
-# Add any Sphinx extension module names here, as strings. They can be extensions
-# coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.doctest', 'sphinx.ext.intersphinx', 'sphinx.ext.todo', 'sphinx.ext.coverage', 'sphinx.ext.pngmath', 'sphinx.ext.ifconfig', 'numpydoc']
-
-
-# Numpydoc generates autosummary directives for all undocumented members. Orange documentation only includes documented
-# member, so _str_member_list is modified to return [] where a list of undocumented members is originally returned.
-numpydoc.docscrape_sphinx.SphinxDocString._str_member_list # if numpydoc changes, this line will cause an error
-numpydoc.docscrape_sphinx.SphinxDocString._str_member_list = lambda x, y : []
+# If your documentation needs a minimal Sphinx version, state it here.
+#needs_sphinx = '1.0'
 
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.intersphinx',
+]
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
 
-# The suffix of source filenames.
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+# source_suffix = ['.rst', '.md']
 source_suffix = '.rst'
 
 # The encoding of source files.
-#source_encoding = 'utf-8'
+#source_encoding = 'utf-8-sig'
 
 # The master toctree document.
 master_doc = 'index'
 
 # General information about the project.
-project = TITLE
-copyright = AUTHOR
+project = 'Orange3-Network'
+copyright = '2016, Biolab'
+author = 'Biolab'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = VERSION
+version = '1.1.2'
 # The full version, including alpha/beta/rc tags.
-release = VERSION
+release = '1.1.2'
```

Comment:
As you already discovered, _conf.py_ is a simple Python module that features [standard Python code](https://github.com/biolab/orange3-network/pull/10/files#diff-684a0ceb2878d48b19612ca360e277efL23).